mariadb.spec

```diff
@@ -1307,6 +1349,18 @@ fi
 %endif
 
 %changelog
+* Tue Jan 10 2017 Michael Bayer <mbayer@redhat.com> - 3:10.1.20-1
+- Update to version 10.1.20
+- Add explicit EVR requirement in main package for -libs
+  Related: #1406320
+- Use correct macro when removing doc files
```

Comment:
I would say this and changelog lines below it do not qualify as end-user documentation and should not be in the %changelog but since those are coming from existing Fedora builds, meh, let's keep them.